nuxt.config.ts

```diff
@@ -47,6 +47,4 @@ const config: NuxtConfig = {
       lang: 'en',
     },
   },
-};
-
-export default config;
+} as NuxtConfig;
```

Comment:
Why do you have to use as here? Would be better to keep the previous version

Reply:
I did that so we're not creating a variable which isn't needed. So if we just default export the object and slap as NuxtConfig it still gets correct typing for linting and autocomplete